Formula/goredo.rb

```diff
@@ -0,0 +1,27 @@
+class Goredo < Formula
+  desc "Go implementation of djb's redo, a Makefile replacement that sucks less"
+  homepage "http://www.goredo.cypherpunks.ru/"
+  url "http://www.goredo.cypherpunks.ru/download/goredo-0.11.0.tar.zst"
+  version "0.11.0"
+  sha256 "4608b75bc8a3dfb62ef305b5eed0f4b28ad39f64ae46baceea8100d90a8666f9"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+  depends_on "zstd" => :build
+
+  def install
+    goredo_prefix = "goredo-#{version}"
+    system "tar", "--use-compress-program", "unzstd", "-xvf", "#{goredo_prefix}.tar.zst"
+    cd goredo_prefix do
+      ENV["GOPATH"] = Pathname.pwd
```

Comment:
This shouldn't be needed with modern Go programs AFAIK.

Reply:
It is when you distribute it with all its dependencies like Sergey has chosen to do.

No need to fetch dependencies from the Internet.

Reply:
https://github.com/golang/go/issues/30329 
> So yes: the current plan is to deprecate GOPATH in Go 1.13 (scheduled for August 2019).

Reply:
TIL! With the help of the author Sergey, we've migrated to Go modules.